src/Controller/MediaController.php

```diff
@@ -45,7 +46,7 @@ public function variation(
             $mediaVariation = $this->converter->getMediaVariation($slug, $variation, $library) ?? throw $this->createNotFoundException('File not found');
             $this->converter->convertMediaVariation($mediaVariation, false);
             $binary = $mediaVariation->getBinary();
-        } catch (\InvalidArgumentException) {
+        } catch (\InvalidArgumentException|VariationNotFoundException) {
             throw $this->createNotFoundException('File not found');
```

Comment:
Keep the previous Exception, important for debug:

```suggestion
        } catch (\InvalidArgumentException|VariationNotFoundException $e) {
            throw $this->createNotFoundException('File not found', $e);
```